app/views/templates/invoices/charge.slim

```diff
@@ -0,0 +1,484 @@
+        .invoice-information-column
+          table.invoice-information-table
+            - if customer.metadata.displayable.any?
+              - customer.metadata.displayable.order(created_at: :asc).each do |metadata|
+                tr
+                  td.body-1 = metadata.key
+                  td.body-2 = metadata.value
+
+      .mb-24.overflow-auto
+        .billing-information-column
+          .body-1 = I18n.t('invoice.bill_from')
+          .body-2
+            - if organization.legal_name.present?
+              | #{organization.legal_name}
+            - else
+              | #{organization.name}
+          .body-2 = organization.address_line1
+          .body-2 = organization.address_line2
+          .body-2
+            span
+              = organization.zipcode
+            - if organization.zipcode.present? && organization.city.present?
+              span
+                | , &nbsp;
+            span
+              = organization.city
+          - if organization.state.present?
+            .body-2 = organization.state
+          .body-2 = ISO3166::Country.new(organization.country)&.common_name
+          .body-2 = organization.email
+        .billing-information-column
+          .body-1 = I18n.t('invoice.bill_to')
+          .body-2
+            - if customer.legal_name.present?
+              | #{customer.legal_name}
+            - else
+              | #{customer.name}
+          .body-2 = customer.address_line1
+          .body-2 = customer.address_line2
+          .body-2
+            span
+              = customer.zipcode
+            - if customer.zipcode.present? && customer.city.present?
+              span
+                | , &nbsp;
+            span
+              = customer.city
+          .body-2 = customer.state
+          .body-2 = ISO3166::Country.new(customer.country)&.common_name
+          .body-2 = customer.email
+
+      .mb-24
+        h2.title-2.mb-4 = total_amount.format(format: I18n.t('money.format'), decimal_mark: I18n.t('money.decimal_mark'), thousands_separator: I18n.t('money.thousands_separator'))
+        .body-1 = I18n.t('invoice.due_date', date: I18n.l(issuing_date, format: :default))
+
+      .invoice-resume.mb-24.overflow-auto
+        table.invoice-resume-table width="100%"
+          tr
+            td.body-2 = I18n.t('invoice.item')
+            td.body-2 style="text-align: left;"
+            td.body-2 style="text-align: left;" = I18n.t('invoice.unit')
+            td.body-2 style="text-align: right;" = I18n.t('invoice.amount')
+          - fees.each do |fee|
+            tr
+              td.body-1 = fee.billable_metric.name
+              td.body-2 style="text-align: left;"
+              td.body-2 style="text-align: left;" = fee.units
+              td.body-2 style="text-align: right;" = fee.amount&.format(format: I18n.t('money.format'), decimal_mark: I18n.t('money.decimal_mark'), thousands_separator: I18n.t('money.thousands_separator'))
```

Comment:
We are repeating `format(format: I18n.t('money.format'), decimal_mark: I18n.t('money.decimal_mark'), thousands_separator: I18n.t('money.thousands_separator'))` all over the templates. I'm wondering if we could find a way to move it in an helper.

Reply:
Added small formatter;)